pkg/host/internal/network/network.go

```diff
@@ -187,6 +190,36 @@ func (n *network) GetNetDevMac(ifaceName string) string {
 	return link.Attrs().HardwareAddr.String()
 }
 
+// GetNetDevNodeGUID returns the network interface node GUID if device is RDMA capable otherwise returns empty string
+func (n *network) GetNetDevNodeGUID(pciAddr string) string {
+	log.Log.V(2).Info("GetNetDevNodeGUID(): get node GUID", "pciAddr", pciAddr)
```

Comment:
This function gets called during the polling of NIC status, for each configured VF. 
I would avoid this logging statement, as it increases config-daemon logs without adding specific value.

You can have a look at the end2end job artifact
https://github.com/k8snetworkplumbingwg/sriov-network-operator/actions/runs/9005014290/artifacts/1484967983

Reply:
Agreed, removed.